towhee/runtime/runtime_pipeline.py

```diff
@@ -49,23 +50,31 @@ def __init__(self,
                  edges: Dict[str, Any],
                  operator_pool: 'OperatorPool',
                  thread_pool: 'ThreadPoolExecutor',
-                 enable_profiler: bool = False):
+                 enable_profiler: bool = False,
+                 enable_tracer: bool = False,
```

Comment:
`enable_tracer` can be removed, we could determine whether `trace_edges`  is empty or not.